### ISSUE TITLE
Fix false positives for vendor-prefixed image-set w/ unit "x" in unit-no-unknown

### ISSUE THREAD

**Issue by srawlins** [stylelint/stylelint#4653]
> Clearly describe the bug

The unit-no-unknown rule reports a false positive when using the "x" unit in a value for a vendor-prefixed image-set ~~property~~ value. E.g.

```css
a { background-image: -webkit-image-set('img-1x.jpg' 1x, 'img-2x.jpg' 2x) }
```

caniuse mentions that `-webkit-image-set` is a valid vendor-prefixed version of `image-set`: https://caniuse.com/#feat=css-image-set

(slight change to "ignore `x` unit in image-set" test case in [lib/rules/unit-no-unknown/__tests__/index.js](https://github.com/stylelint/stylelint/blob/master/lib/rules/unit-no-unknown/__tests__/index.js))

> Which rule, if any, is the bug related to?

`unit-no-unknown`

> What code is needed to reproduce the bug?

```css
a { background-image: -webkit-image-set('img-1x.jpg' 1x, 'img-2x.jpg' 2x) }
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "unit-no-unknown": "always"
  }
}
```

> Which version of stylelint are you using?

13.2.1

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI; `npm run test`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The following warnings were flagged:

```json
{
  "column": 54,
  "line": 1,
  "rule": "unit-no-unknown",
  "severity": "error",
  "text": "Unexpected unknown unit \"x\" (unit-no-unknown)",
}
```

I can write and send a PR.
